src/test/groovy/com/stormpath/tck/AbstractIT.groovy

```diff
@@ -134,6 +146,64 @@ abstract class AbstractIT {
         csrf = ret
     }
 
+    protected void setCSRFAndCookies(requestSpecification, contentType) {
+
+        if (Strings.hasText(csrf) && ContentType.JSON.equals(contentType)) {
+            requestSpecification.header("X-CSRF-TOKEN", csrf)
```

Comment:
@robertjd knows better than me, but should we be also set the `X-XSRF-TOKEN` header here?